### PR TITLE
fix(analyzer): wrong error on missing input file

### PR DESCRIPTION
**Pull request by i-keliukh**
When the passed input file is missing, there were no explicit error
message. Instead, the empty list of files to analyze was passed to the
actual analysis tool. As the result, the error message was incorrect and
hard to understand. Fixed by producing warning on each missing input
file and producing error if the final list is empty.

### CODE REVIEW

**Review by k-dovgan — DISMISSED** [review]

File: universum/analyzers/utils.py [hunk 1]
```diff
@@ -110,7 +110,15 @@ def expand_files_argument(settings: argparse.Namespace) -> None:
     # TODO: subclass argparse.Action
     result = []
```
Comment (o-andrieiev) [inline universum/analyzers/utils.py:111]: why not mark result as List[str]?

Comment (i-keliukh) [reply universum/analyzers/utils.py:111]: BTW, shouldn't it be set instead of list?

Comment (o-andrieiev) [reply universum/analyzers/utils.py:111]: Indeed, set seems like a better fit

Comment (i-keliukh) [reply universum/analyzers/utils.py]: Fixed

**Review by k-dovgan — APPROVED** [review]

**Pull request merged**